lib/services/appmanager-service.ts

```diff
@@ -48,10 +55,49 @@ class AppManagerService implements IAppManagerService {
 
 			this.$logger.info("Successfully uploaded package.");
 
-			var tamUrl = util.format("%s://%s/appbuilder/Services/tam", this.$config.AB_SERVER_PROTO, this.$config.AB_SERVER);
-			this.$logger.info("Go to %s to manage your apps.", tamUrl);
-			this.$opener.open(tamUrl);
+			this.openAppManagerStore();
+		}).future<void>()();
+	}
+
+	public openAppManagerStore(): void {
+		var tamUrl = util.format("%s://%s/appbuilder/Services/tam", this.$config.AB_SERVER_PROTO, this.$config.AB_SERVER);
```

Comment:
We have similar code in `upload` function - https://github.com/Icenium/icenium-cli/blob/master/lib/services/appmanager-service.ts#L51. I suggest to extract it to separate function and reuse the code.